run_tests.py

```diff
@@ -271,7 +271,7 @@ def check_result(status: str, new_part: bool) -> bool:
                             cprint(f'[DBUG]\tpart_pk = {part_pk}')
 
                     # Disable datasheet download/upload after first part (to speed up testing)
-                    settings.DATASHEET_UPLOAD = False
+                    # settings.DATASHEET_UPLOAD = False
```

Comment:
Did you notice the tests to be slow at downloading files? Are we testing the TI URL case added in this PR?

Reply:
@eeintech for now I didn't notice any difference in speed as even without this changed line only the first part is downloading it's attachment. I am not sure why and wanted to checked but it slipped through until now. I suspect it will get very slow with all attachments, so I would only run it for these changes and disable it afterwards again.

Reply:
@eeintech I was stupid and fixed it now. At least the `TPS61221DCKR` in the samples has a TI redirect in digikey